### PR TITLE
Host code for alert

### PR DESCRIPTION
**Pull request by swateeshrivastava**
## Summary
Code changes for host alert.

## Test Plan
Run getAlertLog command for every subsytem and verify the stored alerts details are displaying on the CLI.
Generate alert by giving currentLimit/tempLimit more then default limit and verify run time alert are generate properly.
Clear alert by revert the Limit with default limit and verify alert is cleared.

## Issues
#64

### CODE REVIEW

File: firmware/host/ocmw/ocmw_core.c [hunk 1]
```diff
@@ -765,9 +767,20 @@ int32_t ocmw_send_msg(OCMPMessageFrame ecMsgFrame, uint8_t interface)
...
-            memset(ethRecvBuf, 0, sizeof(ethRecvBuf));
-            ocmw_recv_eth_msgfrom_ec(ethRecvBuf, sizeof(ethRecvBuf), OCMW_EC_STUB_DEV);
-            ocmw_ec_msgparser();
+            if (alertFlag > 0) {
+                while (1) {
+                    memset(ethRecvBuf, 0, sizeof(ethRecvBuf));
+                    ocmw_recv_eth_msgfrom_ec(ethRecvBuf,
+                                sizeof(ethRecvBuf), OCMW_EC_STUB_DEV);
+                    ocmw_ec_msgparser();
+                    if (alertRecord == 0)
```
Comment (arulbose) [inline firmware/host/ocmw/ocmw_core.c]: braces missing for if loop

File: firmware/host/ocwarestub/ocware_stub_database.c [hunk 2]
```diff
@@ -179,6 +181,104 @@ ocware_stub_ret ocware_stub_get_post_database(OCMPMessage *msgFrameData,
...
+    uint8_t printIndex = 0;
+    uint8_t tempAlertCount = allAlertCount;
+
+    OCMPMessageFrame *msgFrame = (OCMPMessageFrame *)buffer;
+    OCMPMessage *msgFrameData = (OCMPMessage*)&msgFrame->message;
+    subSystemNbr = msgFrameData->subsystem;
+    if (subSystemNbr != 0) {
+        for (subsystemIndex = 0; subsystemIndex < allAlertCount;
+            subsystemIndex++) {
+            if (alertGlobalData[subsystemIndex].subsystemId == subSystemNbr)
```
Comment (arulbose) [inline firmware/host/ocwarestub/ocware_stub_database.c]: braces missing

File: firmware/host/log/logFileOccli.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<e> Error: 'recvfrom' [-1-Resource temporarily unavailable]
```
Comment (JoshuaJeyaraj) [inline firmware/host/log/logFileOccli.txt]: kindly remove these log files

File: firmware/utilities/build/build_host_cfg.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+	Host Build
+
```
Comment (JoshuaJeyaraj) [inline firmware/utilities/build/build_host_cfg.txt]: please remove this file

File: firmware/utilities/checksums/hmd5sums.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+bd8d83b75473519480c00496d785a6ec  /home/oc/ocware_alert_host/OpenCellular/firmware/host/bin/occli
+b649d3c0d905fa2a29e8035a494693cd  /home/oc/ocware_alert_host/OpenCellular/firmware/host/bin/ocmw_uart
+6e594743ca4e1bc62eb226fdfde34a08  /home/oc/ocware_alert_host/OpenCellular/firmware/host/bin/ocmw_eth
```
Comment (JoshuaJeyaraj) [inline firmware/utilities/checksums/hmd5sums.txt]: please check and remove files which are not needed

**Pull request merged**